feeder/parsers/FR.py

```diff
@@ -62,5 +62,41 @@ def fetch_production(country_code='FR', session=None):
 
     return data
 
+def fetch_price(country_code, session=None, from_date=None, to_date=None):
+    r = session or requests.session()
+    dt_now=arrow.now(tz='Europe/Paris')
+    formatted_from = from_date or dt_now.replace(days=-1).format('DD/MM/YYYY')
```

Comment:
If you set formatted_from = formatted_to = `dt_now.format('DD/MM/YYYY')` then you actually get the last data point (this is the same behavior as in fetch production. Else it doesn't work (I get the last datapoint of yesterday instead).

Weird API behavior though.